deployment/nginx_chatwoot.conf

```
@@ -20,28 +31,33 @@ server {
   error_log /var/log/nginx/chatwoot_error_443.log;
 
   location / {
-   proxy_pass_header Authorization;
-   proxy_pass http://127.0.0.1:3000;
-   proxy_set_header Upgrade $http_upgrade;
-   proxy_set_header Connection "upgrade";
-   proxy_set_header Host $host;
-   proxy_set_header X-Forwarded-Proto $scheme;
-   proxy_set_header X-Forwarded-Ssl on; # Optional
-   proxy_set_header X-Real-IP $remote_addr;
-   proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-   proxy_http_version 1.1;
-   proxy_set_header Connection "";
-   proxy_buffering off;
-   client_max_body_size 0;
-   proxy_read_timeout 36000s;
-   proxy_redirect off;
+    proxy_pass http://backend;
+    proxy_redirect off;
+
+    proxy_pass_header Authorization;
+    proxy_set_header Host $host;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_set_header X-Forwarded-Ssl on; # Optional
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+    proxy_http_version 1.1;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection $connection_upgrade;
+
+    client_max_body_size 0;
+    proxy_read_timeout 36000s;
   }
 
   ssl_certificate /etc/letsencrypt/live/chatwoot.domain.com/fullchain.pem; # managed by Certbot
   ssl_certificate_key /etc/letsencrypt/live/chatwoot.domain.com/privkey.pem; # managed by Certbot
-  add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
   ssl_protocols TLSv1.2 TLSv1.3;
   ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
-  ssl_prefer_server_ciphers off;
+  ssl_prefer_server_ciphers on;
```

```suggestion
  ssl_prefer_server_ciphers off;
```
Shouldn't this value be `off` in a modern context?

ref: https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=modern&openssl=1.1.1k&guideline=5.6